carbonmark/components/Transaction/Approve.tsx

```diff
@@ -41,12 +42,21 @@ export const Approve: FC<Props> = (props) => {
           success,
         })}
       >
-        <Text>
-          <Trans id="transaction_modal.approve.title">
-            Please confirm the transaction
-          </Trans>
-        </Text>
-        {!!props.description && <Text t="body1">{props.description}</Text>}
+        <div>
+          {props.description && (
+            <FormatTrans
+              id={props.description}
+              style={{ marginBottom: "1.5em" }}
+              links={[
+                {
+                  word: "portfolio",
+                  path: "/portfolio",
+                  blank: true,
```

Comment:
Clever solution, but I think we can make it work with default lingui patterns! This solution might confuse translators who don't know what to do with `link:[]`

My first thought would be to update the props to accept any react element in addition to plain  string, and define them as components to leverage the native lingui react support.

Did you try something similar already?

```jsx
const Description = () => {
  <Text>
    <Trans id="transaction.create_listing.approval_description">
      My text with <Link to="/portfolio">hyperlink</Link> bla bla bla
    </Trans>
  <Text>
}
```
```jsx
<ApprovalText approvalDescription={<Description />} />
```

Reply:
Sounds good. I believe(*) I tried something similar and the text in this method wasn't responding to formatting/line breaks, but I'll try this explicitly so there's not the need for the custom solution.